nautobot/core/views/generic.py

```diff
@@ -255,6 +255,16 @@ def get(self, request):
                 for field_name, values in filter_params.items()
             ]
 
+            if request.GET:
```

Comment:
I see that the `if self.filterset is not None` guard was removed - is that intended/desired?

Reply:
Yes, it was moved to the position where `if self.filterset` was first stated. No need to have two `if self.filterset` statements.